playwrighte2e/pages/citizenHubPage.ts

```diff
@@ -80,7 +80,13 @@ export default class CitizenHubPage extends BasePage {
     supportingMaterialFile:'#supportingMaterialFile',
     uploadFielButton:'#upload',
     copyToOtherPartyYesOrNo:'#copyToOtherPartyYesOrNo',
+<<<<<<< HEAD
+    checkYourAnswerHeading : '//h1[@class="govuk-panel__title"]',
+    responseHeading : '//h2[@class="govuk-summary-list__key govuk-heading-m govuk-!-margin-top-1"]',
+    respondentApplication: '[href="/respondent-applications"]'
+=======
```

Comment:
looks like you've accidentally committed the conflict tags

Reply:
@hsjhita fixed now, thanks